go.mod

```diff
@@ -336,6 +336,10 @@ require (
 
 replace (
 	github.com/CosmWasm/wasmd => github.com/notional-labs/wasmd v0.30.1-0.20230320091624-f5072b9b04a6
+
+	// cosmwasm security patch - cherry
+	github.com/CosmWasm/wasmvm => github.com/CosmWasm/wasmvm v1.2.3
```

Comment:
I think we should check if `notinal` will update the `notional-labs/wasmd` version above. Instead adding additional layer of replace.

Reply:
sure